Makefile

```diff
@@ -252,9 +252,17 @@ sqlc:
 	@$(call print, "Merging SQL migrations into consolidated schemas")
 	go run ./cmd/merge-sql-schemas/main.go
 
-sqlc-check: sqlc
+sqlc-check: 
```

Comment:
Why did you remove the dependency on the `sqlc` target? Now it won't actually generate the schema anymore to see if there's a diff.

Reply:
My bad, missed this. I removed it to verify that the bash fragment I had worked (so I could remove the file, then run the check w/o it adding the file again).